apps/web/pages/signup.tsx

```diff
@@ -130,9 +130,8 @@ export default function Signup({ prepopulateFormValues, token, orgSlug }: Signup
                 <div className="space-y-4">
                   <TextField
                     addOnLeading={
-                      orgSlug
-                        ? getOrgFullDomain(orgSlug, { protocol: true })
-                        : `${process.env.NEXT_PUBLIC_WEBSITE_URL}/`
+                      orgSlug ? getOrgFullDomain(orgSlug, { protocol: false }) : subdomainSuffix()
+
                     }
```

Comment:
can you explain whats going on here?

Reply:
Instead of displaying the entire Gitpod subdomain on the signup page, I used  `subdomainSuffix` in `orgDomains.ts` to get the root domain and display it as shown in the PR description.

Reply:
i would remove this here in this PR and only fix the gitpod file

Reply:
Okay, thanks for taking care of that and merging the PR